UPGRADE.md

```diff
@@ -4,6 +4,14 @@
 
 ### Listbuilder
 
+The listbuilder uses now expressions to build the query. In course of these changes some default values have been
+removed from some methodes of the `AbstractListBuilder` because of unclear meaning / effect. Changed function parameters:
+
+- where (conjunction removed)
+- between (conjunction removed)
+
+### Listbuilder
```

Comment:
Please find better headings for both of these, having two times the same heading seems strange.